src/Changes/ResolvedChanges.php

```diff
@@ -64,8 +64,17 @@ public function getNormal() {
           $sequence_id = array_search($uuid, array_column($all_sequences, 'entity_uuid'));
 
           if ($sequence_id) {
-            $changes[$uuid] = $this->buildChangeRecord($all_sequences[$sequence_id], $entity);
+            // The sequence is already in range.
+            $additional_sequence = $all_sequences[$sequence_id];
           }
+          else {
+            // We add a new sequence for the non-existing entity.
+            $this->sequenceIndex->add($entity);
```

Comment:
I think this problematic as it will lead to the creation of lots of new sequences. The more clients replicate, the more sequences will be create and the longer initial replication will last.

Instead, I think we can just send multiple change entries with the same sequence number to the client. The sequence id can be seen as a "global database version" id, so I think it's ok to have multiple change entries for it as long as they are delivered as part of the same get-changes request, what they are.